app/components/blacklight/facet_item_pivot_component.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module Blacklight
+  # Render facet items and any subtree
+  class FacetItemPivotComponent < ::ViewComponent::Base
+    def self.id_counter
+      @id_counter ||= 0
+      @id_counter += 1
+    end
+
+    with_collection_parameter :facet_item
+
+    def initialize(facet_item:, wrapping_element: 'li', suppress_link: false, collapsing: nil)
+      @facet_item = facet_item
+      @wrapping_element = wrapping_element
+      @suppress_link = suppress_link
+      @collapsing = collapsing.nil? ? facet_item.facet_config.collapsing : collapsing
+      @id = "h-#{self.class.id_counter.to_s(36)}" if @collapsing
+    end
+
+    def call
+      facet = Blacklight::FacetItemComponent.new(facet_item: @facet_item, wrapping_element: nil, suppress_link: @suppress_link)
+
+      content_tag @wrapping_element, role: 'treeitem' do
+        concat facet_toggle_button if has_items? && @collapsing
+        concat content_tag('span', render_component(facet), class: "facet-values #{'facet-leaf-node' if has_items? && @collapsing}", id: @id && "#{@id}_label")
+
+        if has_items?
+          concat(content_tag('ul', class: "pivot-facet list-unstyled #{'collapse' if @collapsing}", id: @id, role: 'group') do
+            render_component(
+              self.class.with_collection(
+                @facet_item.items.map { |i| facet_item_presenter(i) }
+              )
+            )
+          end)
+        end
+      end
+    end
+
+    private
+
+    def has_items?
+      @facet_item.items.present?
+    end
+
+    def facet_toggle_button
+      content_tag 'button', class: 'btn facet-toggle-handle collapsed',
+                            data: { toggle: 'collapse', target: "##{@id}" },
+                            aria: { expanded: false, controls: @id, describedby: "#{@id}_label" } do
+        concat content_tag 'span', t('blacklight.search.facets.pivot.show_html'), class: 'show'
```

Comment:
I'm really not a big fan of putting html in i18n. Is there a reason we can't put just the text of the  .sr-only element in the i18n file?